### PR TITLE
Navbar added

### LINKED ISSUE #7: Implement Navigation.jsx
image
Location: rEvery page
Number of Images: 1
React Icon: no
Link: each button should link to the corresponding page for the website
Path: \src\components\NavBar.jsx
Description: component includes text for each page with a corresponding link, club icon on the left side, buttons should lower opacity when hovering over them, navbar should stick to the top of the page when the user scrolls down

Do not use React Bootstrap
[Example](https://github.com/acm-ucr/ucr-swim-website/blob/dev/src/components/NavBar.jsx)

### PR DESCRIPTION
**Pull request by Fuyuki325**
Desktop:
image

Mobile:
image

### CODE REVIEW

**Comment by Fuyuki325** [comment]
when i click the figma, it says the font is Gurajada but there is no Gurajada in the tailwind config

File: src/components/Navigation.jsx [hunk 1]
```diff
@@ -1,3 +1,110 @@
...
+  },
+  {
+    link: "involve",
+    name: "Involve",
+  },
+  {
+    link: "events",
+    name: "Events",
+  },
+];
```
Comment (seanquiambao) [inline src/components/Navigation.jsx]: create a data directory and put this in `Navigation.js` file.

File: src/components/Navigation.jsx [hunk 2]
```diff
@@ -1,3 +1,110 @@
...
+
+        {/* mobile menu */}
+        <div
+          className={
+            nav
+              ? "transition ease-in-out md:hidden flex flex-col items-center justify-evenly w-[100%] duration-500 bg-biscuits-peach space-y-2 pb-2"
+              : "hidden"
+          }
+        >
+          {items.map((item, index) => (
```
Comment (seanquiambao) [inline src/components/Navigation.jsx:62]: image
on mobile view, maybe make the text a little bigger.

**Review by seanquiambao — CHANGES_REQUESTED** [review]
made some changes, but after those comments and using guarjada font this should be good to go!

**Comment by Fuyuki325** [comment]
image
added gurajada and made text bigger

**Review by seanquiambao — APPROVED** [review]

**Pull request merged**